packages/charts/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -31,12 +31,11 @@ export class Arc extends React.Component<IArcProps, IArcState> {
   }
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33447&commitId=bc2f7abb351b1cf667150b475fe17194caadb014&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>